src/components/dashboard/account/AccountSettings.js

```diff
@@ -1,17 +1,17 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 
 import PaymentSettingsContainer from './PaymentSettingsContainer';
 import UserSettingsContainer from './UserSettingsContainer';
 
 class AccountSettings extends Component {
     render() {
         return (
-            <div>
+            <div className="content h-100 min-h-100 relative overflow-y-auto pt4 pb7">
```

Comment:
There is no general CSS file yet.  I was planning on going through all of the files to do it on my branch (as best as possible)!

Do we even need these classes, though?  After removing the top nav from the dashboard, I wonder if we even need all of this.  I remember it being the reason for a lot of complications.  I know for sure `pb7` was added to avoid it.